tests/acl/test_acl_l3v4v6.py

```diff
@@ -0,0 +1,1224 @@
+LOG_EXPECT_ACL_RULE_CREATE_RE = ".*Successfully created ACL rule.*"
+LOG_EXPECT_ACL_RULE_REMOVE_RE = ".*Successfully deleted ACL rule.*"
+
+PACKETS_COUNT = "packets_count"
+BYTES_COUNT = "bytes_count"
+
+
+@pytest.fixture(scope="module", autouse=True)
+def remove_dataacl_table(duthosts):
+    """
+    Remove DATAACL to free TCAM resources.
+    The change is written to configdb as we don't want DATAACL recovered after reboot
+    """
+    TABLE_NAME = "DATAACL"
+    for duthost in duthosts:
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME))['stdout_lines']
+        data_acl_existing = False
+        for line in lines:
+            if TABLE_NAME in line:
+                data_acl_existing = True
+                break
+        if data_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME))
+            cmds = [
+                "config acl remove table {}".format(TABLE_NAME),
+                "config save -y"
+            ]
+            duthost.shell_cmds(cmds=cmds)
+    yield
+    # Recover DUT by reloading minigraph
+    for duthost in duthosts:
+        config_reload(duthost, config_source="minigraph")
+
+
+def get_t2_info(duthosts, tbinfo):
+    # Get the list of upstream/downstream ports
+    downstream_ports, upstream_ports, acl_table_ports_per_dut = defaultdict(list), defaultdict(list), defaultdict(list)
+    upstream_port_id_to_router_mac_map, downstream_port_id_to_router_mac_map = {}, {}
+    downstream_port_ids, upstream_port_ids = [], []
+    port_channels = dict()
+
+    for duthost in duthosts:
+        if duthost.is_supervisor_node():
+            continue
+        upstream_ports_per_dut, downstream_ports_per_dut, acl_table_ports = (defaultdict(list),
+                                                                             defaultdict(list), defaultdict(list))
+
+        for sonic_host_or_asic_inst in duthost.get_sonic_host_and_frontend_asic_instance():
+            namespace = sonic_host_or_asic_inst.namespace if hasattr(sonic_host_or_asic_inst, 'namespace') \
+                  else DEFAULT_NAMESPACE
+            if duthost.sonichost.is_multi_asic and namespace == DEFAULT_NAMESPACE:
+                continue
+            asic_id = duthost.get_asic_id_from_namespace(namespace)
+            router_mac = duthost.asic_instance(asic_id).get_router_mac()
+            mg_facts = duthost.get_extended_minigraph_facts(tbinfo, namespace)
+            for interface, neighbor in list(mg_facts["minigraph_neighbors"].items()):
+                port_id = mg_facts["minigraph_ptf_indices"][interface]
+                if "T1" in neighbor["name"]:
+                    downstream_ports_per_dut[namespace].append(interface)
+                    downstream_port_ids.append(port_id)
+                    downstream_port_id_to_router_mac_map[port_id] = router_mac
+                elif "T3" in neighbor["name"]:
```

Comment:
Since the test is expected to run on `T0` and `T1`, this check is probably not needed.